CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 
 ## Unreleased
```

Comment:
Shouldn't this be changed to `## v0.3.2` or are these changes for the upcoming version already?